components/StatusFilter.tsx

```diff
@@ -20,12 +20,12 @@ interface StatusAction {
 }
 
 interface StatusFilterProps {
-  handleChecks: (value: string) => void;
+  handleApplyStatusFilterCheck: (value: string) => void;
```

Comment:
Check는 함수명에서 제거되도 괜찮을 것 같아요~

```suggestion
  handleApplyStatusFilter: (value: string) => void;
```